lutris/sysoptions.py

```diff
@@ -227,6 +227,16 @@ def get_vk_icd_choices():
         "advanced": True,
         "help": _("Switch to US keyboard QWERTY layout while game is running"),
     },
+    {
+        "option": "nvidia_separate_glshaderdiskcache",
+        "type": "bool",
+        "default": True,
+        "label": _("Separate GL Shader Disk Caches (NVIDIA)"),
+        "advanced": True,
+        "help": _("Use per-game GL shader disk caches for NVidia hardware. "
```

Comment:
same request but here it would be better if the tooltip was "Use a dedicated location for a game's shader cache generated by the NVIDIA driver.". that's just imo tho